lib/foreman/renderer/configuration.rb

```diff
@@ -42,7 +42,11 @@ class Configuration
         :number_with_delimiter,
         :number_with_precision,
         :number_to_human_size,
-        :gem_version_compare
+        :gem_version_compare,
+        :parse_yaml,
+        :parse_json,
+        :to_json,
+        :to_yaml,
```

Comment:
Bad rubocop! Looks like it trips up on the combined lines like `log_*`

Reply:
I'll revert it to 
```suggestion
        :to_yaml
```
Maybe it would be a good idea to change `Style/TrailingCommaInArrayLiteral` cop to `consistent_comma`?

Reply:
:+1: for `consistent_coma`

Reply:
I vote REMOVING this cop completely.

Reply:
I strongly insist we enforce trailing comma's because it avoids modifying multiple lines when all you want to do is add one. The fact it didn't want to add one here is IMHO a problem.